sdks/python/apache_beam/io/hadoopfilesystem.py

```diff
@@ -399,6 +418,26 @@ def checksum(self, url):
         file_checksum[_FILE_CHECKSUM_BYTES],
     )
 
+  def metadata(self, url):
+    """Fetch metadata fields of a file on the FileSystem.
+
+    Args:
+      url: string url of a file.
+
+    Returns:
+      :class:`~apache_beam.io.filesystem.FileMetadata`.
+      Note: last_updated field is not supported yet.
```

Comment:
is it not supported or supported? I see we're passing the parameter to `FileMatadata` in line 439?

Reply:
It is supported since revision of [01f639f](https://github.com/apache/beam/pull/17380/commits/01f639f67a273f1e35c1207ccfdea59eba0a4551). Outdated comments. my fault.